src/applications/gi/containers/LicenseCertificationSearch.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import LicenseCertificationSearchFields from '../components/LicenseCertificationSearchFields';
+
+function LicenseCertificationSearch() {
+  return (
+    <div>
+      <section className="lc-wrapper vads-u-padding-x--2p5 small-screen:vads-u-padding-x--2">
```

Comment:
Platform is updating breakpoint prefixes - all but medium-screen are being renamed. In this case `small-screen` should be replaced with `mobile-lg` (More info in the canvas in platform-design-system on slack)

Reply:
Thanks for the tip! The suggested changes have been made.